content/asp.net/getting-started/views.md

```diff
@@ -1,20 +1,121 @@
+
+The intent of the Razor view engine is to make switching between HTML and c# as seamless as possible in order to render output that is useful to the client. We can create our document tags as we would in a normal HTML document and use the `@` character to indicate that we are making use of c#. As our view is essentially a class, it's also easy to understand how Razor can expose data and keywords as though they were properties and methods. Here is an example taken from the sample project in `Views\Person\Index.cshtml`:
+
+```html
+@model MvcBasics.Models.Person
+
+<h1>Person Details</h1>
+<p>@Model.FirstName @Model.LastName</p>
+```
+
+To better understand how the above code works, let's take a step back to see how the runtime arrives at our view equipped with the knowledge that is needed in order to render something useful from that code. We'll come back to the above syntax shortly.
+
+## The Mechanics of a View
+
+The primary function of the `ViewImports.cshtml` is to provide a mechanism for us to pull in any namespaces that we want available to all our views. This facilitates bringing in a namespace that contains your view models, rather than having to declare them in each of your views (or reference those classes explicitly by their fully qualified name). The other thing you can do here is to manage the tag helpers that  Razor is aware of either individually or by namespace. 
+
+`ViewStart.cshtml` is semantically different from `ViewImports.cshtml` in that it can be used as a starting point for view-related logic. You typically use it to set a default layout for you views, but you can also use it to change the layout based on a user's role or to inspect and manipulate the request. A word of caution here, however, for it is as easy to over-reach inside of the `ViewImports` as it is in a view and to start writing business logic in the wrong places.  `ViewImports` is likely not the best place to call out to services or enforce business rules.
+
+Finally, the `_Layout.cshtml` is used as a template for rendering your web pages and is located in the `Views\Shared` folder of your project.  Layouts allow you to define mandatory and optional sections of what will become your rendered page that can be later filled in by your views. A default layout is created in the project template complete with common styles and page head information, a navigation bar, a footer, a section stubbed in for the body of the page as well as an optional section to add scripts to the final output in your views.  You can have as many layouts as are needed, perhaps to enable specific functionality for a subset of users or to provide a different experience in an administration area in your site. 
```

Comment:
Perhaps end this section with a link to learn more: 
https://docs.microsoft.com/en-us/aspnet/core/mvc/views/layout